deploy/podman/flightctl-db/flightctl-db-config/enable-superuser.sh

```
@@ -33,4 +33,87 @@ else
     echo "POSTGRESQL_MASTER_USER not set, skipping superuser configuration"
 fi
 
+echo "Setting up FlightCtl database users for production deployment..."
+
+# Get passwords from environment variables (set by podman secrets)
+: "${POSTGRESQL_PASSWORD:?POSTGRESQL_PASSWORD must be set}"
+DB_APP_PASSWORD="${POSTGRESQL_PASSWORD}"
+
+# Database user setup - using the admin user we just configured
+DB_HOST="localhost"
```

Shellcheck (0.10.0)

[warning] DB_HOST appears unused. Verify use (or export if used externally).

(SC2034)